airguardian-core/battery-application/src/main/kotlin/systems/ajax/motrechko/airguardian/batteryapplication/application/port/DroneChargingApplicationProducerOutPort.kt

```diff
@@ -0,0 +1,8 @@
+package systems.ajax.motrechko.airguardian.batteryapplication.application.port
+
+import reactor.core.publisher.Mono
+import systems.ajax.motrechko.airguardian.commonresponse.application.drone_battery_charging_application.proto.DroneBatteryChargingApplication
```

Comment:
`DroneBatteryChargingApplication` is DTO that is part of infrastructure layer. Application layer should not depend on infrastructure layer